app/views/search/index.html.erb

```diff
@@ -55,8 +55,14 @@
               <%= l(:label_full_text_search_order_target_score) %>
             </label>
             <label>
-              <%= form.radio_button "order_target", "date", name: "order_target" %>
-              <%= l(:label_full_text_search_order_target_date) %>
+              <!-- TODO: 'date' is deprecated but maintained for backward compatibility. -->
+              <!-- Use 'last_modified_time' or 'registered_time' explicitly. -->
+              <%= form.radio_button "order_target", "last_modified_time", checked: ["date", "last_modified_time"].include?(@search_request.order_target), name: "order_target" %>
```

Comment:
Could you fold a long line for readability?

```suggestion
              <%= form.radio_button "order_target",
                                    "last_modified_time",
                                    checked: ["date", "last_modified_time"].include?(@search_request.order_target),
                                    name: "order_target" %>
```

Reply:
fix: [afb81f1](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/afb81f1759727aeb1a541181ff3d8c18201aec8d)